cran-comments.md

```diff
@@ -1,8 +1,12 @@
-## bioRad 0.7.3 
+## bioRad 0.8.0 
```

Comment:
Review carefully all closed issues since last release (https://github.com/adokter/bioRad/issues?q=is%3Aissue+is%3Aclosed) and document them here, I think several are missing. Also please reference the issue number or pull request at the end of the line describing each new feature of fix (see previous releases for example)